src/scss/_base.scss

```diff
@@ -101,6 +101,7 @@ h6,
 %h6 {
   @extend %hN;
   font-size: $font-size-h6;
+  text-transform: uppercase;
```

Comment:
This will make all `<h6>` uppercase right?

Reply:
Yes!